CHANGELOG.md

```diff
@@ -103,7 +103,7 @@ This version is marked by a major version bump because of the significant upgrad
 
 ### Add
 
-- Add [SEP-1](https://stellar.org/protocol/sep-1) fields to `StellarTomlResolver` for type checks ([#794](https://github.com/stellar/js-stellar-sdk/pull/794)).
```

Comment:
should we be retro-changing release notes from past? maybe leave as-is and add a item to `Unreleased` that mentions this rename of TomlResolver